packages/mjml-core/src/index.js

```diff
@@ -26,7 +26,7 @@ export default function mjml2html(mjml, options = {}) {
   let errors = []
 
   if(typeof options.skeleton === 'string') {
-    options.skeleton = (opt) => require(options.skeleton)
+    options.skeleton = require(options.skeleton)
```

Comment:
Mhh, only issue is the require will be relative to the file calling it, might need to pass the process.cwd if it's a relative path ?